cspell.yaml

```diff
@@ -146,6 +146,15 @@ ignorePaths:
   - .editorconfig
   - .github/CODEOWNERS
   - packages/samples/test/output/**
+  - "**/BenchmarkDotNet.Artifacts/**"
+  - cspell.yaml
+overrides:
+  - filename: "packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/TypeFormatterTests.cs"
```

Comment:
can you not have `cspell:ignore` in comment instead?

https://cspell.org/configuration/document-settings/#enable--disable-checking-sections-of-code

Reply:
merged before I saw this https://github.com/microsoft/typespec/pull/3607